app/controllers/decidim/translation_addons/admin/reports_controller.rb

```diff
@@ -3,12 +3,83 @@
 module Decidim
   module TranslationAddons
     module Admin
-      # Controller used to manage participatory process types for the current
+      # Controller used to manage translation reports for the current
       # organization
       class ReportsController < Decidim::TranslationAddons::Admin::ApplicationController
+        helper Decidim::PaginateHelper
+        helper Decidim::ResourceHelper
+        include Decidim::TranslationAddons::Admin::Reports::Filterable
+
+        layout "decidim/admin/global_moderations"
+        before_action :set_translation_breadcrumb_item
+
         def index
+          @reports = filtered_collection
+        end
+
+        def unreport
+          report = Decidim::TranslationAddons::Report.find params[:id]
+          Decidim::TranslationAddons::Unreport.call(report, current_user) do
+            on(:ok) do
+              flash[:notice] = I18n.t("unreport.success", scope: "decidim.admin")
+              redirect_to reports_path
+            end
+
+            on(:invalid) do
+              flash[:alert] = I18n.t("unreport.invalid", scope: "decidim.admin")
+              redirect_to reports_path
+            end
+          end
+        end
+
+        def request_translation
+          report = Decidim::TranslationAddons::Report.find params[:id]
+          Decidim::TranslationAddons::RequestTranslation.call(report, current_user) do
+            on(:ok) do
+              flash[:notice] = I18n.t("translation_request.success", scope: "decidim.admin")
+              redirect_to reports_path
+            end
+
+            on(:not_missing) do
+              flash[:alert] = I18n.t("decidim.shared.notification_messages.not_missing")
+              redirect_to reports_path
+            end
+
+            on(:missing_default_locale) do
+              flash[:alert] = I18n.t("translation_request.missing_default_locale", scope: "decidim.admin")
+              redirect_to reports_path
+            end
+
+            on(:invalid) do
+              flash[:alert] = I18n.t("translation_request.invalid", scope: "decidim.admin")
+              redirect_to reports_path
+            end
+          end
+        end
+
+        def configure
```

Comment:
not sure what it does, where is supposed to be used, but if this is a method designed for being called on report's details, then i would extract 
```
def report 
  @report ||= Decidim::TranslationAddons::Report.find params[:id]
end

def configure 
  @report_details = report.details.find_by(decidim_translation_addons_report_id: params[:id]) 
end 
```